TriblerGUI/tribler_request_manager.py

```diff
@@ -224,6 +224,10 @@ def __init__(self, window=None):
         self.status_code = -1
         self.on_cancel = lambda: None
 
+    def __del__(self):
+        if self.reply:
+            self.reply.deleteLater()
```

Comment:
This can also be externally managed (we learned this the hard way): `deleteLater` should always be called in a `try` `except RuntimeError` block.